daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_dev/daml_lf_1.proto

```diff
@@ -269,6 +269,10 @@ enum PrimType {
   // *Available in versions >= 1.3*
   MAP = 16;
 
+  // Builtin type 'TGenMap`
+  // *Available in versions >= 1.dev*
+  GENMAP = 20;
```

Comment:
I suggest we move this down below `TYPE_REP` to keep the numbers in order.

Reply:
fixed